src/routes/superAdmin.js

```diff
@@ -0,0 +1,21 @@
+import { lazy } from 'react';
+import { Route, Switch } from 'react-router-dom';
+
+const SuperAdmins = lazy(() => import('Views/SuperAdmin/SAManagement'));
+const FormSuperAdmin = lazy(() => import('Views/SuperAdmin/SAManagement/Form'));
+const Admins = lazy(() => import('Views/SuperAdmin/AdminManagement'));
+const AdminForm = lazy(() => import('Views/SuperAdmin/AdminManagement/Form'));
+
+const SuperAdminsRoutes = () => {
+  return (
+    <Switch>
+      <Route exact path="/superAdmins/admins" component={Admins} />
+      <Route exact path="/superAdmin/admin/form" component={AdminForm} />
```

Comment:
this should be superAdmins/admins in plural

Reply:
done